profiler/src/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ set(PROFILER_SOURCES
     profile_grouped_conv_bwd_data.cpp
     profile_conv_tensor_rearrange.cpp
     profile_transpose.cpp
+    profile_permute_scale.cpp
```

Comment:
You have to move this below line `if(DTYPES MATCHES "fp16" OR NOT DEFINED DTYPES)`

Reply:
I think it will disable tests for fp32.